### ISSUE TITLE
Switcher Output in Shortcode Module

### ISSUE THREAD

**Issue by Tekanewa** [Codestar/codestar-framework#415]
When using the "switcher" option within the shortcode framework, the generated shortcode will only output the switcher value when it is set to "yes"/"on"/"true", but will totally ignore the switcher value and drop the associated shortcode attribute when the switcher is set to "no"/"off"/"false", which should not be the case.

Simply dropping a shortcode attribute just because the switcher is set to "no"/"off"/"false" is the wrong behavior; the attribute should still be generated, just with a value of "0" (zero).

It would also be great if the switcher option could be set to actually return/output a "true" or "false", instead of "1" or "0".

**Comment by Codestar** [comment]
Hmm, noted this, i will think solutions about this. ;)

**Comment by Tekanewa** [comment]
First of, great to see you back! I was growing slightly afraid that this project was getting abandoned, as there haven't been any replies in a while. I have already incorporated CS into two of my plugins and am generally very happy with it. :)

For now, I created my own custom switcher element, using your's as base, which is outputting "true/false" instead of "1/0" and also correctly outputting the attribute if the value is in fact set to "false". But it would of course be better if the native switcher element could do that. :)
